bin/update-gh-pages

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -eu
+
+repo_uri="https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
+remote_name="origin"
+main_branch="${DEPLOY_BRANCH}"
+target_branch="gh-pages"
+build_dir="dist"
+
+cd "$GITHUB_WORKSPACE"
+
+git config user.name "$GITHUB_ACTOR"
+git config user.email "${GITHUB_ACTOR}@bots.github.com"
+
+git checkout "$target_branch"
+#git rebase "${remote_name}/${main_branch}"
```

Comment:
Don't we want rebase here? We want to have the latest changes from main to be included in gh-pages before deploying?

Reply:
No. Since this build job is the only source of changes to the `gh-pages` branch.
So a "force push" makes sure, that we always have the most recent deployment pushed :-)